Formula/p/portaudio.rb

```diff
@@ -13,7 +13,7 @@ class Portaudio < Formula
     regex(/href=.*?pa[._-]stable[._-]v?(\d+)(?:[._-]\d+)?\.t/i)
     strategy :page_match do |page, regex|
       # Modify filename version (190700) to match formula version (19.7.0)
-      page.scan(regex).map { |match| match&.first&.scan(/\d{2}/)&.map(&:to_i)&.join(".") }
+      page.scan(regex).map { |match| match.first.scan(/\d{2}/).map(&:to_i).join(".") }
```

Comment:
The nil-forwarding here should be unnecessary cc @samford (tested with current page)

Reply:
```suggestion
      page.scan(regex).map { |match| match[0].scan(/\d{2}/).map(&:to_i).join(".") }
```

Yep, `match` will always be present in this context when the regex matches. Similarly, the capture group isn't optional, so `match[0]` will also be present when the regex matches.

I recently cleaned up similar instances in homebrew/cask, removing unnecessary safe navigation operators and using numeric addressing (i.e., `match.first` -> `match[0]`). I'll create a follow-up PR to address the same for homebrew/core in a bit.

Reply:
Works for me (it's a better (typesafe) to use `.fetch` if non-nil is guaranteed, though that can be a bit pedantic)